site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -216,26 +242,598 @@ They can do the same for Chrome on desktop by going to
+  status](https://fedidcg.github.io/FedCM/#the-idp-sign-in-status-api) to
+  mitigate the issue.
+* **Sign in to IdP API**: To support [various
+  scenarios](https://github.com/fedidcg/FedCM/issues/348), when a user is not
+  signed in to the IdP, the browser provides a UI for the user to sign in
+  without leaving the RP.
+
+Finally, there are things we believe still need to be done, based on feedback
+from
+[Mozila](https://github.com/mozilla/standards-positions/issues/618#issuecomment-1221964677),
+[Apple](https://lists.webkit.org/pipermail/webkit-dev/2022-March/032162.html)
+and [TAG
+reviewers](https://github.com/w3ctag/design-reviews/issues/718#issue-1165654549).
+We are working to evaluate the best solution for these open questions:
+
+* **Improving user comprehension and matching intent**: As [Mozilla
+  noted](https://github.com/mozilla/standards-positions/issues/618#issuecomment-1221964677),
+  we’d like to continue exploring different UX formulations and surface areas,
+  as well as triggering criteria.
+* **Identity Attributes and Selective Disclosure**: As our [TAG Reviewers
+  noted](https://github.com/w3ctag/design-reviews/issues/718#issuecomment-1171733526),
+  we’d like to provide a mechanism to selectively share more or less identity
+  attributes (such as emails, age brackets, phone numbers, and so on).
+* **Raising the Privacy Properties**: As Mozilla suggested
+  [here](https://github.com/mozilla/standards-positions/issues/618#issuecomment-1221964677),
+  we’d like to continue exploring  mechanisms to offer better privacy
+  guarantees, such as IdP blindness, directed identifiers.
+* **Relationship with WebAuthn**: As suggested by
+  [Apple](https://lists.webkit.org/pipermail/webkit-dev/2022-March/032162.html),
+  we are super excited to see the progress on
+  [passkeys](http://goo.gle/passkeys) and to work on providing a coherent and
+  cohesive experience between FedCM, Passwords, WebAuthn and WebOTP.
+* **Login Status**: As Apple suggested with the Privacy CG’s [Login Status
+  API](https://github.com/privacycg/is-logged-in), we share the intuition that
+  the user’s login status is a useful bit of information that can help browsers
+  make informed decisions, and we are excited to see what opportunities arise
+  from that.
+* **Enterprises and Education**: As is clear at the FedID CG, there are still [a
+  lot of use
+  cases](https://github.com/fedidcg/use-case-library/blob/main/decision_tree_flows/login/Federated%20Login%20OIDC%20Oauth2%20Auth%20Code%20Flow.png)
+  that are not well served by FedCM that we’d like to work on, such as  
+  front-channel logout (the ability for an IdP to send a signal to RPs to
+  logout) and support for SAML.
+* **Relationship with mDLs/VCs/etc**: continue working to understand how these
+  fit within FedCM, for example with the [Mobile Document Request
+  API](https://github.com/WICG/mobile-document-request-api).
+
+## How can identity providers develop FedCM?
```

Comment:
This section lacks a lot of things an IDP would need to do in order to use FedCM. In particular, the requirements mentioned are RP requirements, no? This could be renamed maybe to "What are requirements on sites to invoke the FedCM API?"

Reply:
Originally this section had origin trial registration instructions but they are removed and the leftover is not suitable for the title. I thought of moving this close to the RP's sign-in API part, but thought it's not suitable. Let me try some changes so it suits better. Changing the title to "How can you develop FedCM?".